src/locales/default.json

```diff
@@ -279,7 +281,20 @@
     },
     "hideSpace": "Hide space from homepage",
     "links": "Social accounts",
-    "subspaces": "Sub-Spaces",
+    "subspaces": {
+      "label": "Sub-spaces",
+      "information": "Sub-spaces will only be shown once configured on both the main space and the sub-space(s). {docs}",
+      "parent": {
+        "label": "Main space",
+        "placeholder": "e.g. fabien.eth",
+        "information": "The main space of your DAO, which this space is a sub-space of. It will be displayed in the space's sidebar menu."
```

Comment:
```suggestion
        "information": "The space that this space is a sub-space of will be displayed in the sidebar menu"
```

Reply:
I like. I just don't get why we're not using full-stops to end a sentence anymore. I've seen this in other places to.